src/wasm.rs

```diff
@@ -32,18 +31,16 @@ use crate::pfsys::Snarkbytes;
 /// Generate circuit params in browser
 #[wasm_bindgen]
 pub fn gen_circuit_params_wasm(
```

Comment:
Don't we have to change the parameters here to accept a calibration folder? Or will that be included in run args?